### ISSUE TITLE
Unsupported nodes for supported layers but on torch.nn.functional

### ISSUE THREAD

**Issue by DavidNeveziStrango** [eml-eda/plinio#12]
Hello.

I come again to you with an additional bug i have noticed.
During my work, i have found out that simply taking a model off-shelf from the torch-hub, vision section, at the function call of PIT (when converting a Seed model into it's PIT version), it has difficulties with layers that have not been initialized as a class, but simply used as function in the forward pass (ex. torch.nn.functional.adaptive_avg_pool2d).

You may try to reproduce the bug with the following notebook (and eventually the pip package)
[examples/coco.ipynb](https://github.com/DavidNeveziStrango/ubervvald/blob/main/examples/coco.ipynb)

For the 13th cell, when initiating the class, you may comment it and add the following one: 
```model = torch.hub.load("pytorch/vision:v0.10.0", "densenet121", weights=False, num_classes=len(classes))```
you may change the designated architecture with anything that has in the official pytorch implementation the functional version of AdaptiveAvgpool2d (other examples could be resnext50_32x3d, inception_v3). 

Pytorch v 2.6.0+

Thanks in advance!

**Comment by danielepagliari** [comment]
Hi David,

Sorry for the very late reply.
Unfortunately, to our knowledge, there's no easy way to support in a single "shot" both nn.Modules and equivalent functions from the functional API. Torch.fx sees them as two separate ops, so each must be supported individually. If you would like to help us support functional versions of AvgPool or other ops, please feel free to submit a PR.
Thanks for all your useful comments and contributions!

Daniele
